mutiny-core/src/nodemanager.rs

```diff
@@ -539,6 +539,8 @@ pub struct Plan {
     pub amount_sat: u64,
 }
 
+type CurrencyHashMap = Option<HashMap<String, (f32, Duration)>>;
```

Comment:
You don't need to make a `type` definition here and it doesn't need to be an `Option`. `None` here is the same as an empty `HashMap` so may as well just use `HashMap`